firehose/parsers/flawfinder.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python
+#
+#   Copyright 2017 David Carlos  <ddavidcarlos1392@gmail.com>
+#   This library is free software; you can redistribute it and/or
+#   modify it under the terms of the GNU Lesser General Public
+#   License as published by the Free Software Foundation; either
+#   version 2.1 of the License, or (at your option) any later version.
+#
+#   This library is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#   Lesser General Public License for more details.
+#
+#   You should have received a copy of the GNU Lesser General Public
+#   License along with this library; if not, write to the Free Software
+#   Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301
+#   USA
+
+import sys
+import re
+from subprocess import check_output
+from firehose.model import Message, Point, \
+    File, Location, Generator, Metadata, Analysis, Issue
+
+
+def main():
+    """ Main entry to flawfinder parser """
+    arg_file = sys.argv[1]
+    report = load_file(arg_file)
+    analysis = parse_file(report)
+    sys.stdout.write(str(analysis.to_xml()))
+    sys.stdout.write('\n')
+
+def load_file(arg_file):
+    """ Loads the Flawfinder report in memory
+
+    :arg1: file passed by cli
+    :returns: file loaded in memory
+    """
```

Comment:
This doesn't actually load a file into memory; it merely returns a ``file`` object representing a file opened for reading.  The code would be clearer if you got rid of this function and replace users with just the ``open`` builtin.